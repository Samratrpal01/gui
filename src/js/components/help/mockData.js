export const helpProps = {
  docsVersion: undefined,
  isHosted: false,
  menderVersion: 'master',
  isEnterprise: true,
  tenantCapabilities: {},
  tokens: [],
  versions: {
    repos: {
      mender: '3.3.0',
      'mender-artifact': '3.8.0',
      'mender-binary-delta': '1.3.1',
      'mender-cli': '1.8.0',
      'mender-configure-module': '1.0.4',
      'mender-connect': '2.0.2',
      'mender-convert': '3.0.0',
      'mender-gateway': '1.0.0',
      'monitor-client': '1.2.0'
    },
    releaseDate: '01-01-2003'
  }
};
