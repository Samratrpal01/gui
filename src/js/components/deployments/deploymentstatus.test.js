import React from 'react';

import { undefineds } from '../../../../tests/mockData';
import { render } from '../../../../tests/setupTests';
import DeploymentStatus from './deploymentstatus';

describe('DeploymentStatus Component', () => {
  it('renders correctly', async () => {
    const { baseElement } = render(<DeploymentStatus refreshStatus={jest.fn()} />);
    const view = baseElement.firstChild.firstChild;
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});
