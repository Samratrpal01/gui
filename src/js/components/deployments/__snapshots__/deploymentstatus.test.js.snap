// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeploymentStatus Component renders correctly 1`] = `
.emotion-0 {
  -webkit-column-gap: 8px;
  column-gap: 8px;
  display: grid;
  grid-template-columns: repeat(auto-fit, 32px);
}

.emotion-0>div {
  -webkit-column-gap: 4px;
  column-gap: 4px;
}

.emotion-0 .disabled {
  opacity: 0.1;
}

<div
  class="flexbox emotion-0 "
>
  <div
    aria-label="Skipped"
    class="flexbox centered disabled"
    data-mui-internal-clone-element="true"
  >
    <img
      src="skipped_status.png"
    />
    <div
      class="status"
    >
      0
    </div>
  </div>
  <div
    aria-label="Pending"
    class="flexbox centered disabled"
    data-mui-internal-clone-element="true"
  >
    <img
      src="pending_status.png"
    />
    <div
      class="status"
    >
      0
    </div>
  </div>
  <div
    aria-label="In progress"
    class="flexbox centered disabled"
    data-mui-internal-clone-element="true"
  >
    <img
      src="progress_status.png"
    />
    <div
      class="status"
    >
      0
    </div>
  </div>
  <div
    aria-label="Successful"
    class="flexbox centered disabled"
    data-mui-internal-clone-element="true"
  >
    <img
      src="success_status.png"
    />
    <div
      class="status"
    >
      0
    </div>
  </div>
  <div
    aria-label="Failed"
    class="flexbox centered disabled"
    data-mui-internal-clone-element="true"
  >
    <img
      src="error_status.png"
    />
    <div
      class="status"
    >
      0
    </div>
  </div>
</div>
`;
